Composer/plugins/vacore/src/index.ts

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+import path from 'path';
+
+export default async (composer: any): Promise<void> => {
+  if (process.env.VA_CREATION) {
+    // register the base template which will appear in the new bot modal
+    composer.addBotTemplate({
+      id: 'va-core',
+      name: 'VA Core',
+      description: 'The core of your new VA - ready to run, just add skills!',
```

Comment:
we need to localize these strings using formatMessage

Reply:
Okay, I have added formatMessage